debian/changelog

```diff
@@ -1,3 +1,48 @@
+cloud-init (21.2-3-g899bfaa9-0ubuntu1~18.04.1) UNRELEASED; urgency=medium
```

Comment:
s/UNRELEASED/bionic/

We also appear to have modified a bunch of older changelog entries (to strip trailing whitespace, presumably thanks to your editor): let's drop those as needless churn.

Reply:
Updated...and woe to those who save trailing whitespace! :wink: